src/manifolds/SymmetricPositiveDefinite.jl

```diff
@@ -128,6 +276,109 @@ function default_estimation_method(::SymmetricPositiveDefinite, ::typeof(mean))
     return GeodesicInterpolation()
 end
 
+@doc raw"""
+    convert(::Type{AbstractMatrix}, p::SPDPoint)
+
+return the point `p` as a matrix.
+The matrix is either stored within the [`SPDPoint`](@ref) or reconstructed from `p.eigen`.
+"""
+convert(::Type{AbstractMatrix}, p::SPDPoint)
+convert(::Type{AbstractMatrix}, p::SPDPoint) = p.p
```

Comment:
```suggestion
"""
convert(::Type{AbstractMatrix}, p::SPDPoint) = p.p
```
That one line isn't useful